src/web/routes/application/check/get.js

```diff
@@ -38,6 +31,17 @@ const getRowData = (req) => (step) => {
   return Array.isArray(result) ? result.map(applyPathToRow) : applyPathToRow(result)
 }
 
+// a step is navigable if it hasn't defined an isNavigable function.
+const stepIsNavigable = (lastStep, session) => !lastStep.hasOwnProperty('isNavigable') || lastStep.isNavigable(session)
```

Comment:
I would change `lastStep` -> `step` in the parameters as this function doesn't really have a concept of `last`

Reply:
Updated to step.